src/pages/Report.tsx

```diff
@@ -0,0 +1,213 @@
+                    <li>seja curiosa por natureza;</li>
+                    <li>perfil mão na massa e ágil;</li>
+                    <li>sabe ouvir e se comunicar;</li>
+                    <li>busca e tenha um histórico de aprendizado contínuo;</li>
+                    <li>
+                      busca e fomenta a colaboração, mas também tenha as suas
+                      contribuições individuais inovadoras;
+                    </li>
+                    <li>
+                      entenda o objetivo e propósito da empresa, e consiga atuar
+                      com autonomia dentro de suas responsabilidades.
+                    </li>
+                  </ul>
+                  <p className="font-medium  text-gray-600 my-3">
+                    {" "}
+                    Responsabilidades e atribuições:
+                  </p>
+                </div>
+              </div>
+              <div className="bg-[#FCFCFC] border rounded-lg mt-6 py-4 px-6 flex items-center justify-between hidden md:flex">
+                <div>
+                  <h4 className="font-semibold  text-gray-600">
+                    Contate o suporte SouJunior
+                  </h4>
+                  <Link to="/suporte" className="underline text-blue-500">
+                    https://www.soujunior/suporte
+                  </Link>
+                  <p className="text-gray-600">soujunior@suporte.com</p>
+                </div>
+                <div>
+                  <img src={mailIcon} alt="Mail Icon" />
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="flex flex-1 border md:px-6 py-7 px-3 rounded-md flex-col">
+            <h4 className="font-bold md:text-2xl text-xl mb-2 md:mb-1 text-gray-para text-gray-600 tracking-wide">
+              Por que está denúnciando está vaga?
+            </h4>
+            <p className="font-normal text-sm mb-4">
+              Informe abaixo a informação mais relevante para a denuncia da
+              vaga!
+            </p>
+            <hr />
+            <div className="relative mb-8">
+              <textarea
+                name="message_report"
+                id=""
+                placeholder="Descreva a causa da denúncia?"
+                className="px-4 border py-4 rounded-lg mt-5 min-h-[527px] resize-none w-full "
+                value={message}
+                onChange={handleMessage}
+                maxLength={MAX_LENGTH}
+              ></textarea>
+              <span className="text-sm text-gray-600 absolute bottom-4 right-4    ">
+                {count}/{MAX_LENGTH}
+              </span>
+            </div>
+            <div className="w-full md:flex">
+              <button className="bg-red-600 md:flex md:flex-1 md:w-auto w-full hover:bg-red-700  text-white py-3 px-6 font-semibold rounded md:mb-0 mb-4 md:mb-0 flex flex-1 justify-center items-center md:mr-4 mr-0">
+                Denúnciar
```

Comment:
Denunciar não possui acento!
```suggestion
                Denunciar
```